packages/contentstack-dev-dependencies/src/spy.ts

```diff
@@ -37,6 +37,6 @@ export function spy<T, K extends keyof T>(object: T, path: K, prefix?: string) {
   };
 }
 
-export const fancy = test.register('spy', spy) as unknown as TestWitSpyType;
+// export const fancy = test.register('spy', spy) as unknown as TestWitSpyType;
```

Comment:
@cs-raj , remove unnecessary commented code

Reply:
Hi @aman19K this will be removed once all the test cases are finalised as I might be trying to add spy on fancy for the other plugins